bool manual_mode_override
bool start_task
bool cancel_task
bool pause_task
bool continue_task
bool emergency_stop
